app/javascript/mastodon/models/account.ts

```diff
@@ -56,6 +56,8 @@ export interface AccountShape
   note_plain: string | null;
   hidden: boolean;
   moved: string | null;
+  avatar_description: string | null;
+  header_description: string | null;
```

Comment:
Then you'd not need string | null as the type — an empty string is as food as null in this case: there is no description to show.